app/components/pagination/bs-pagination.vue

```diff
@@ -68,6 +68,10 @@ const props = defineProps({
 
 const changePage = (val) => {
   emit("update:change", val);
+  const eltewt = document.getElementsByClassName('orca-lti-content-item')[0];
```

Comment:
please add additiioinal check for the case when `document.getElementsByClassName('orca-lti-content-item')` returns an empty list or undefined:

```js
  elements = document.getElementsByClassName('orca-lti-content-item');
  if (elements && elements != null && elements != undefined && elements.size() > 0) {
    eletewt = elements[0];
 ...
  }
```